app/PostScreen.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { supabase } from '@/supabase/client';
+import '../assets/global.css';
```

Comment:
I think you can remove this import, it might introduce some weird errors with styling since its importing all the global styles here, but they aren't being used. If a class name from the global css file conflicts with the local style.module.css, then the styling can be weird and its hard to debug.